app/views/curation_concerns/base/_form.html.erb

```diff
@@ -34,7 +34,7 @@
       </div>
 
       <div id="savewidget" class="col-xs-12 col-sm-4 fixedsticky" role="complementary">
-        <%= render 'form_progress', f: f %>
+        <%= render 'form_progress', f: f, agreement_accepted: @form.agreement_accepted %>
```

Comment:
Now that you've made it an accessor on the form it shouldn't be a problem. However, there's still no need to pass it as a separate local.